examples/2Dturbulence_multigrid/esel.cxx

```diff
@@ -113,7 +113,7 @@ class ESEL : public PhysicsModel {
       Field3D vort2 = D2DX2(phi) + D2DZ2(phi) + DDX(N) * DDX(phi) + DDZ(N) * DDZ(phi);
       vort_error = (vort - vort2);
 
-      dump.write();
+      //      dump.write();
```

Comment:
Following up on @dschwoerer's comment on the previous PR, I think just removing `test_laplacian` and its uses entirely makes the most sense.

This whole physics model is maybe worth getting rid of entirely anyway